### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by TS1RR** [TeamNewPipe/NewPipe#11772] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Hey,
i am looking for a feature, wich i saw and often use at Spotify or Vanced. The sleeptimer-function. Maybe there is a chance to implement this into the next releases?
Hope to hear from you soon. 
Kind regards from Germany 
Flo

**Comment by asandikci** [comment]
See: #2830
